src/test/java/AttendanceTracker/AppTest.java

```diff
@@ -4,11 +4,9 @@
 package AttendanceTracker;
 
 import org.junit.jupiter.api.Test;
-import static org.junit.jupiter.api.Assertions.*;
```

Comment:
next time you test let's leave this line, so we can have the full testing suite.